tests/test_enterprise/api/test_views.py

```diff
@@ -2857,6 +2851,7 @@ def test_bulk_enrollment_in_bulk_courses(
 
         permission = Permission.objects.get(name='Can add Enterprise Customer')
         self.user.user_permissions.add(permission)
+        mock_get_course_mode.return_value = 'verified'
```

Comment:
small ask to make the 'verified' a test file level constant

Reply:
or import from somewhere if it exists

Reply:
I added it to the file 😄